src/z3c/autoinclude/utils.py

```diff
@@ -89,9 +90,9 @@ def distributionForDottedName(package_dottedname):
 
     if len(valid_dists_for_package) == 0:
         raise LookupError("No distributions found for package `%s`; are you sure it is importable?" % package_dottedname)
-    
+
     if len(valid_dists_for_package) > 1:
-        non_namespaced_dists = filter(lambda x: len(x[1]) is 0, valid_dists_for_package)
+        non_namespaced_dists = list(filter(lambda x: len(x[1]) is 0, valid_dists_for_package))
```

Comment:
Ugh.  I suppose it works, but rewriting as a comprehension would be better.  If you can figure out what is going on here ;)